pkg/kapp/resourcesmisc/custom_waiting_resource.go

```diff
@@ -80,5 +91,9 @@ func (s CustomWaitingResource) IsDoneApplying() DoneApplyState {
 		}
 	}
 
+	if condObsGenMatchWait {
+		return DoneApplyState{Done: false, Message: fmt.Sprintf(
+			"Waiting for generation %d to be observed by status condition(s)", obj.Metadata.Generation)}
```

Comment:
Just a suggestion, would love to know what others think.
```suggestion
			"Condition(s) waiting for generation %d to be observed", obj.Metadata.Generation)}
```

Reply:
As we all agreed that we want to go with older change only. Closing this without change.